phpcs.xml.dist

```diff
@@ -192,6 +196,9 @@
 
     <rule ref="Squiz.Commenting.FunctionComment.WrongStyle">
         <!-- https://github.com/squizlabs/PHP_CodeSniffer/issues/1961 -->
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/DatabasePlatformMock.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/DriverMock.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/SequenceMock.php</exclude-pattern>
```

Comment:
Is this because of `/* MOCK API */`? Does the WrongStyle error happen with `// MOCK API ` too?

Reply:
Yes and yes